modules/mono/glue/GodotSharp/GodotSharp/Core/Vector2.cs

```diff
@@ -600,7 +617,7 @@ internal void Normalize()
         }
 
         /// <summary>
-        /// Returns this vector with each component snapped to the nearest multiple of <paramref name="step"/>.
+        /// Returns a new vector with each component snapped to the nearest multiple of the corresponding component in <paramref name="step"/>.
```

Comment:
The old sentence meant "this but snapped", if we are saying it's a new vector we need to reference that it is based on the old one. Of course that should be obvious, but it should still be stated in the doc text somehow. Maybe "Returns a copy of this vector [...]"?

Reply:
This is used pretty much everywhere else, I'd say that's for a dedicated PR adding a new unified style, this just unifies the C# and GDScript docs